src/encoded/static/components/award.js

```diff
@@ -129,29 +128,9 @@ function createDoughnutChart(chartId, values, labels, colors, baseSearchUri, nav
 
 
 // Display and handle clicks in the chart of labs.
-const LabChart = React.createClass({
```

Comment:
That’s a future goal. The main thing stopping it right now is Computational Predictions which have _just_ enough different to make that a non-obvious case so far. I’m sure there’s a way, but I haven’t had a chance to really nail that down yet.